tests/test_ecs_databag_timer.h

```diff
@@ -49,10 +49,10 @@ TEST_CASE("[Modules][ECS] Test basic timer databag operations.") {
 		CHECK(td.is_done(handle) == false);
 		CHECK(td.get_remaining_seconds(handle) >= (1.0 - CMP_EPSILON));
 
-		td.set_now(2.5 * 1'000'000.0);
+		td.internal_set_now(2.5 * 1'000'000.0);
 		CHECK(td.get_remaining_seconds(handle) >= (0.5 - CMP_EPSILON));
 
-		td.set_now(4 * 1'000'000.0);
+		td.internal_set_now(4 * 1'000'000.0);
 		CHECK(td.is_done(handle));
 	}
```

Comment:
I would add some tests to verify the pause mechanism, so we are sure that it works now and forever, even after new features will be added to this.

Reply:
Yep, making unit tests is what divides humans between the lazy and the smart. Marking as WIP until they are done.